app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/FaceDetectionPage.xaml.cs

```diff
@@ -51,14 +51,19 @@ namespace Microsoft.ProjectOxford.Face.Controls
     /// </summary>
     public partial class FaceDetectionPage : Page, INotifyPropertyChanged
     {
-
+        
```

Comment:
If this has not been merged to upstream master when you see this commit, fix it :)

Reply:
Done. Seems it has already been merged...